routers/user/oauth.go

```diff
@@ -571,3 +631,16 @@ func handleAuthorizeError(ctx *context.Context, authErr AuthorizeError, redirect
 	redirect.RawQuery = q.Encode()
 	ctx.Redirect(redirect.String(), 302)
 }
+
+func handleBearerTokenError(ctx *context.Context, beErr BearerTokenError) {
+	ctx.Resp.Header().Set("WWW-Authenticate", fmt.Sprintf("Bearer realm=\"\", error=\"%s\", error_description=\"%s\"", beErr.ErrorCode, beErr.ErrorDescription))
+	if beErr.ErrorCode == BearerTokenErrorCodeInvalidRequest {
```

Comment:
Use switch is better

Reply:
Implemented switch (came from Python, there is no switch available - my fault..)
Added also a default for unknown ErrorCode (this would be a server error, because it should never be reached, only if there is some internal failure)